tests/test_utils.py

```diff
@@ -237,11 +264,30 @@ def keyring_missing(monkeypatch):
     monkeypatch.delitem(sys.modules, 'keyring', raising=False)
 
 
+@pytest.fixture
+def keyring_missing_get_credentials(monkeypatch):
+    """
+    Simulate that 'import keyring' raises an ImportError
```

Comment:
This docstring needs to be updated.

Reply:
So does the name that gets deleted immediately below... guess that's why the coverage didn't hit 100%?